### PR TITLE
events-page v-1.1

### PR DESCRIPTION
**Pull request by Dhruvil-Lakhtaria**
Events page layout and design.
View @ - https://dhruvil-lakhtaria.github.io/incident-2022/
We can keep font for heading same everywhere - Shizuru (can be linked to html via google fonts)

image

Font for content yet to be decided for events page
//TODO - 
Add popup screen for each event
Add content for each event

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  left: 50%;
+  width: 100%;
+  height: auto;
+  -webkit-transform: translateX(-50%) translateZ(0);
+  transform: translateX(-50%) translateZ(0);
+  -webkit-animation: confetti linear infinite;
+  animation: confetti linear infinite;
+}
+
+img:nth-child(1) {
```
Comment (AdarshNaidu) [inline css/index.css]: Let's add a class to these styles, so it doesn't effect other images.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1200px;
+  margin-block: 2rem;
+  gap: 2rem;
+}
+.events-card__image{
+  height: 15em;
+}
+img {
```
Comment (AdarshNaidu) [inline css/index.css]: Here as well!

File: css/index.css (hunk 1 above)
Comment (Dhruvil-Lakhtaria) [reply css/index.css]: Done

File: css/index.css (hunk 2 above)
Comment (Dhruvil-Lakhtaria) [reply css/index.css]: Done

**Pull request merged**